raiden-cli/src/custom.d.ts

```diff
@@ -0,0 +1,9 @@
+import { RaidenChannel } from 'raiden-ts';
+
+declare global {
```

Comment:
Would this better go on a `global.d.ts` file? For consistency

Reply:
:heavy_checkmark: